sources/packages/backend/workflow/src/workflow-definitions/parttime-configure-disbursement.bpmn

```diff
@@ -108,7 +108,9 @@
+          <zeebe:output source="={&#10;  CSLPAmount: round half up(finalFederalAwardNetCSLPAmount / 2, 0),&#10;  CSGPAmount: round half up(finalFederalAwardNetCSGPAmount / 2, 0),&#10;  CSGDAmount: round half up(finalFederalAwardNetCSGDAmount / 2, 0),&#10;  CSPTAmount: round half up(finalFederalAwardNetCSPTAmount / 2, 0),&#10;  BCAGAmount: round half up(finalProvincialAwardNetBCAGAmount / 2, 0),&#10;  SBSDAmount: round half up(finalProvincialAwardNetSBSDAmount / 2, 0)&#10;}" target="firstHalfDisbursementValues" />
```

Comment:
Following the naming convention from the rest of the source code, the acronym in the beginning would be lowercase, as below.
```
{
  cslpAmount: round half up(finalFederalAwardNetCSLPAmount / 2, 0),
  csgpAmount: round half up(finalFederalAwardNetCSGPAmount / 2, 0),
  csgdAmount: round half up(finalFederalAwardNetCSGDAmount / 2, 0),
  csptAmount: round half up(finalFederalAwardNetCSPTAmount / 2, 0),
  bcagAmount: round half up(finalProvincialAwardNetBCAGAmount / 2, 0),
  sbsdAmount: round half up(finalProvincialAwardNetSBSDAmount / 2, 0)
}
```

Reply:
Updated.